tests/arns.test.mjs

```diff
@@ -263,6 +263,24 @@ describe('ArNS', async () => {
     assert.equal(tokenCost, 600000000);
   });
 
+  it('should get registration prices', async () => {
+    const priceListResult = await handle({
+      Tags: [
+        { name: 'Action', value: 'Get-Registration-Prices' },
+        { name: 'Intent', value: 'Buy-Record' },
```

Comment:
wondering if we need intent - and should it be `GET-REGISTRATION-FEES`

Reply:
Removed the intent and changes to Get-Registration-Fees